app/decorators/guest_decorator.rb

```diff
@@ -0,0 +1,13 @@
+class GuestDecorator
+  def lockout_time_remaining_in_words
```

Comment:
I think this should be tested / changed -- @monfresh suggested that we capture `second_factor_locked_at` from the user before we log them out. Just wasn't sure where this logic should go? Ideally we could somehow share with the logic in `UserDecorator` so the methods can stay in sync.... suggestions?